forge-gui-mobile/src/forge/adventure/data/BiomeStructureData.java

```diff
@@ -0,0 +1,68 @@
+package forge.adventure.data;
+
+import java.awt.image.BufferedImage;
```

Comment:
Does this class is only accessed via the desktop adventure editor? mobile version (android) don't have BufferedImage

Reply:
sadly it is used by the api of the WaveLengthCollapse algo.
Then I have to copy yoink it and copy it to the project to rewrite it...